tools/version.props.template

```diff
@@ -0,0 +1,14 @@
+<Project>
```

Comment:
This file contains the common NuGet package metadata. It gets renamed to `version.props` during the build process.